database/migrations/2018_12_21_155421_populate_departments.php

```diff
@@ -14,39 +14,39 @@ class PopulateDepartments extends Migration
     public function up()
     {
         DB::table('departments')->insert([
-            ['id' => 1, 'name' => 'treasury_board'],
```

Comment:
This is to avoid screwing up the auto-increment index, right? But we have to be careful, this edited version will never run on prod, since this migration has already ran.

Reply:
We'll need to reset the auto-increment either manually, or with a new migration.

Reply:
Why do we need to reset the auto increment? Are we referencing departments by ID somewhere?

Reply:
I think this is resolved.

Reply:
We have departments with ids up to 12, but the auto-increment index is still at 1. So here's the issue: when someone tries to add a new department from the admin portal, it will fail with a "Duplicate key" error.